challenges/2021_cloud_base_height/05_optuna_tuning.ipynb

```diff
@@ -0,0 +1,923 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    searcher = OptunaSearch(metric=["val_loss_mean"], mode=["min"])
</pre><p>are we actually using optuna, I thought you weren't in the end?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
I was able to use the optuna integration built into ray for nicer hparam search